google/cloud/security/scanner/audit/buckets_rules_engine.py

```diff
@@ -0,0 +1,255 @@
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rules engine for Bucket acls"""
+from collections import namedtuple
+import itertools
+import re
+import sys
+
+# pylint: disable=line-too-long
+from google.cloud.security.common.gcp_type.bucket_access_controls import BucketAccessControls
```

Comment:
import module name instead of class name (now that I think about it, I realize that we haven't been totally consistent either... trying to get there to that point too)